### PR TITLE
HIVE-22626. Fix Replication related tests.

### PR DESCRIPTION
**Pull request by ayushtkn**

### CODE REVIEW

File: itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/BaseReplicationAcrossInstances.java [hunk 1]
```diff
@@ -59,7 +59,7 @@ static void internalBeforeClassSetup(Map<String, String> overrides, Class clazz)
     conf.set("hive.repl.cmrootdir", "/tmp/");
     conf.set("dfs.namenode.acls.enabled", "true");
     MiniDFSCluster miniDFSCluster =
-        new MiniDFSCluster.Builder(conf).numDataNodes(1).format(true).build();
+        new MiniDFSCluster.Builder(conf).numDataNodes(2).format(true).build();
```
Comment (ArkoSharma) [inline itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/BaseReplicationAcrossInstances.java:62]: Needs to be done in 'internalBeforeClassSetupExclusiveReplica' function too.

File: itests/hive-unit/src/test/java/org/apache/hadoop/hive/metastore/TestReplChangeManager.java [hunk 2]
```diff
@@ -95,7 +95,7 @@ private static void internalSetUpProvidePerm() throws Exception {
     String noPermBaseDir = Files.createTempDirectory("noPerm").toFile().getAbsolutePath();
     configuration.set(MiniDFSCluster.HDFS_MINIDFS_BASEDIR, noPermBaseDir);
     configuration.set("dfs.client.use.datanode.hostname", "true");
-    permDdfs = new MiniDFSCluster.Builder(configuration).numDataNodes(1).format(true).build();
+    permDdfs = new MiniDFSCluster.Builder(configuration).numDataNodes(2).format(true).build();
```
Comment (ArkoSharma) [inline itests/hive-unit/src/test/java/org/apache/hadoop/hive/metastore/TestReplChangeManager.java:98]: Also requires same in 'internalSetUp' function.

File: ql/src/java/org/apache/hadoop/hive/ql/parse/repl/ReplLogger.java [hunk 3]
```diff
@@ -1,4 +1,4 @@
-/*
+/**
```
Comment (ArkoSharma) [inline ql/src/java/org/apache/hadoop/hive/ql/parse/repl/ReplLogger.java:1]: typo

**Review by ArkoSharma — CHANGES_REQUESTED** [review]

**Review by ArkoSharma — APPROVED** [review]

**Comment by ayushtkn** [comment]
Enables 3 tests:

**TestStatsReplicationScenariosMMNoAutogather:**
http://ci.hive.apache.org/job/hive-flaky-check/308/testReport/org.apache.hadoop.hive.ql.parse/

**TestStatsReplicationScenariosACIDNoAutogather:**
http://ci.hive.apache.org/job/hive-flaky-check/304/testReport/org.apache.hadoop.hive.ql.parse/

**TestStatsReplicationScenariosACID:**
http://ci.hive.apache.org/job/hive-flaky-check/305/testReport/org.apache.hadoop.hive.ql.parse/

**Pull request merged**